Dockerfile

```diff
@@ -4,6 +4,8 @@ COPY . .
 RUN make
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+WORKDIR /go/src/github.com/openshift/cluster-logging-operator
```

Comment:
we have two `WORKDIR` entries in this Dockerfile

Reply:
I wasnt certain if WORKDIR applies to seperately